ajax/libs/tuesday/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tuesday",
+  "filename": "tuesday.min.css",
+  "author": "Hansol Kim",
```

Comment:
Let's modify this field as the same with upstream.
https://github.com/shakrmedia/tuesday/blob/master/package.json#L20-L23
It's good to same with upstream, except some situation.

Thanks

Reply:
Well, I did notice this part, but my mentor told me to use the author's name instead of that one you said. Which one should I follow?

Reply:
@ralphchung 
Ummm...
Maybe there are some reason from upstream that they want to use organization name as author field.
Here are some cases https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/react/package.json#L34, https://github.com/cdnjs/cdnjs/pull/11647/files.
We know that no one named `Facebook`, it's company name but it in author field of package.json.

Although, the author field in package.json be needed as the same with upstream by us, in commit message, we need to cc @author instead of organization name, to notice real person(@organization_name can't be noticed).

What do you think?

Thanks.

Reply:
@sufuf3 
I think using the organization name is preferable just like what you said.
I will start following your instructions above to pass the CI test.

Thanks a lot.

Reply:
Sorry, I gave you the wrong answer.
I thought you were asking should you cc author or organization. That's why my answer was "organization". We should just follow the author info from the upstream. 
If there is no author info in package.json. You might need to open an issue to check what author info is correct.
Thank you both.